Core/src/org/sleuthkit/autopsy/casemodule/Bundle.properties-MERGED

```diff
@@ -354,6 +354,7 @@ UnpackagePortableCaseProgressDialog.title.text=Unpackage Portable Case Progress
 UnpackageWorker.doInBackground.canceled=Unpackaging canceled by user
 UnpackageWorker.doInBackground.errorCompressingCase=Error unpackaging case
 UnpackageWorker.doInBackground.errorFinding7zip=Could not locate 7-Zip executable
+UnpackageWorker.doInBackground.previousSeenCase=Case {0} has been previously opened do you want to open it again?
```

Comment:
This is out of sync with the Bundle messages string.

Reply:
Pushed up file